config/taxonomy/catalog.structs.schema.json

```diff
@@ -51,20 +55,42 @@
                                 "type": "string",
                                 "description": "name of column or component"
                             },
-                            "tags": { "$ref": "catalog.values.schema.json#/definitions/valid_metadata"}
+                            "tags": { "$ref": "#/definitions/tags" }
                         },
-                        "required":["name"]
+                        "required": ["name"]
                     }
                 }
             },
-            "required":["name"],
-            "additionalProperties": false
-        }
-    },
+            "required": ["name"],
+            "additionalProperties": {
+                "creationTime": {
+                    "type": "string",
+                    "description": "Time when the asset was created"
+                },
+                "CreatedBy": {
```

Comment:
All the other properties start with a small letter, so I assume this one should as well

Reply:
@ritwikc please remove the entire additionalProperties here. We discussed it last time and I was going to leave it for the next iteration but obviously it adds unnecessary overhead in this PR.

Reply:
Let me correct it.

Reply:
Done